client/.env.example

```diff
@@ -0,0 +1 @@
+REACT_APP_API_URL=http://localhost:3001
```

Comment:
Because we are using the proxy in package.json, we don't need this and we won't need to add CORS support for the back-end just yet 

The proxy already routes API requests to localhost:3001